src/Mod/Sketcher/App/Sketch.cpp

```diff
@@ -5244,23 +5225,6 @@ TopoShape Sketch::toShape() const
     TopoShape result;
     std::vector<GeoDef>::const_iterator it = Geoms.begin();
 
-#if 0

```

Comment:
IIRC, a similar change was earlier rejected for some reason. @chennes, I think you had described why, could you confirm?

Reply:
I think this is OK to remove: there's no comment explaining the commented-out nature of the code.